From c9fe0229521fd6eba8a1d2e925aa9e1e4bca3293 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:46 -0400
Subject: [alsa] hda: Fix build error in hda_tegra.c

Message-id: <140972823573-372-patchset-jkysela@redhat.com>
Patchwork-id: 91068
O-Subject: [RHEL 7.1 PATCH 371/442] ALSA: hda - Fix build error in hda_tegra.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The "list" field has been omitted from struct azx, but its
initialization remained mistakenly in hda_tegra.c, which leads to a
compile error:
sound/pci/hda/hda_tegra.c: In function 'hda_tegra_create':
sound/pci/hda/hda_tegra.c:481:22: error: 'struct azx' has no member
named 'list'

Reported-by: kbuild test robot <fengguang.wu@intel.com>
Fixes: 9a34af4a3327 ('ALSA: hda - Move more PCI-controller-specific stuff from generic code')
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 178942b69f4eb695bfb92742b6082052d6730ca0)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_tegra.c b/sound/pci/hda/hda_tegra.c
index cf69daf..cc4a923 100644
--- a/sound/pci/hda/hda_tegra.c
+++ b/sound/pci/hda/hda_tegra.c
@@ -453,7 +453,6 @@ static int hda_tegra_create(struct snd_card *card,
  chip->driver_type = driver_caps & 0xff;
  chip->dev_index = 0;
  INIT_LIST_HEAD(&chip->pcm_list);
- INIT_LIST_HEAD(&chip->list);
 
  chip->codec_probe_mask = -1;
 
-- 
1.7.1